workspaces/web/src/media/sound.js

```diff
@@ -12,4 +12,12 @@ const sound = {
     }),
 }
 
+export const playAudio = (type,filename) => {
+    if(!window.Cypress){
```

Comment:
ok, this is perfect, because this never had a preloading system before anyway. Perhaps, we can create a task for building the preloading system, but it's not relevant here! :+1: